tests/pipelines/test_pipelines_summarization.py

```diff
@@ -54,7 +55,7 @@ def run_pipeline_test(self, summarizer, _):
         )
         self.assertEqual(outputs, [{"summary_text": ANY(str)}])
 
-        if not isinstance(model.config, (T5Config, LongT5Config, LEDConfig)):
+        if not isinstance(model.config, (SwitchTransformersConfig, T5Config, LongT5Config, LEDConfig)):
```

Comment:
Modify the comment below a well :-)

Reply:
Great! Fixed in 7c3e5aabf6cd72b84e24af34d377fc1310cdca8f